src/include/sof/drivers/dw-dma.h

```diff
@@ -118,7 +118,7 @@
 #define DW_DMA_CFG_TRIES	10000
 
 /* channel drain timeout in microseconds */
-#define DW_DMA_TIMEOUT	1333
+#define DW_DMA_TIMEOUT	600
```

Comment:
I wonder if we should be more clever here. We have 2 use cases for the timeout.
1) DAI copies - limited to period size, we should calculate this at runtime e.g. 1ms period should timeout at 1.1ms ?
2) Host copies - typically very fast, I've witmessed 1ms period copied on BYT in around 20us. We could say 100uS here for host copies ?